src/deploy/functions/containerCleaner.ts

```diff
@@ -128,6 +133,55 @@ export class ContainerRegistryCleaner {
   }
 }
 
+// top level function for purging artifacts
+export async function purgeArtifacts(projectId: string, location?: string) {
+  // purge Container Registry
+  const purger = new ContainerRegistryPurger();
+  await purger.purge(projectId, location);
+
+  // TODO: purge Artifact Registry
+}
+
+export class ContainerRegistryPurger {
```

Comment:
I don't think we should make this a class - theres no constructor, and there's only one real method, so this seems like it would be more readable as a plain old function.

Reply:
Changed it to an exportable function. Added a default parameter for DockerHelper for unit testing